students/1st/myfullname/styles/login.css

```
@@ -0,0 +1,61 @@
+@import url('https://fonts.googleapis.com/css2?family=Dancing+Script:wght@700&display=swap');
+
+* {
+box-sizing: border-box;}
```

중괄호 닫는 위치나 탭 간격 등을 신경써주세요 :)
```suggestion
* {
  box-sizing: border-box;
}
```